src/libraries/System.Formats.Tar/tests/TarReader/TarReader.File.Async.Tests.cs

```diff
@@ -263,7 +263,6 @@ public async Task AllowSpacesInOctalFieldsAsync(string folderName, string testCa
         [InlineData("invalid-go17")] // Many octal fields are all zero chars
         [InlineData("issue11169")] // Checksum with null in the middle
         [InlineData("issue10968")] // Garbage chars
-        [InlineData("writer-big")] // The size field contains an euro char
```

Comment:
Why was this removed?

Reply:
The size field of `writer-big` is a valid GNU numeric format. The issue with the file is that it doesn't include all data for that large file size, which throws as the reader moves across the data, but not when reading the header (which is what the test is doing).

Reply:
This is one of those files that we found in a golang tar reader and decided to bring it here to ensure we had a clearly documented behavior in our reader. This case had an unexpected char in one of the fields so we _expect_ our tar reader to throw.

So based on your description, GNU can advance a little more than the other formats, but the file is _still_ malformed even for GNU. If you re-add the test case, does it still throw InvalidDataException? If yes, then I think we should bring that test case back. If not, then I think we should duplicate this test method just for that test case and make sure we capture the exception that is thrown.

Reply:
This file is a header that describes a 16GB file. Previously it failed because the size was considered invalid.
Now it fails because we're trying to seek past the length.

> If not, then I think we should duplicate this test method just for that test case and make sure we capture the exception that is thrown.

The test would be something like this:

```cs
        [Fact]
        public void Throw_ArchiveIsShort()
        {
            // writer-big has a header for a 16G file but not its contents.
            using MemoryStream archiveStream = GetTarMemoryStream(CompressionMethod.Uncompressed, "golang_tar", "writer-big");
            using TarReader reader = new TarReader(archiveStream);
            // MemoryStream throws when we try to change its Position past its length.
            Assert.Throws<ArgumentOutOfRangeException>(() => reader.GetNextEntry());
        }
```

Should I add it?

Reply:
Yes please! Let's not lose coverage of it.